app/src/main/res/layout/activity_quiz.xml

```diff
@@ -27,9 +27,21 @@
     <androidx.fragment.app.FragmentContainerView
         android:id="@+id/question_details_container"
         android:layout_width="0dp"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         app:layout_constraintStart_toEndOf="@+id/separator"
```

Comment:
I think that you need to constraint the bottom of this container view if you want the layout to be more consistent.

Reply:
Yes